Documentation/ceph-upgrade.md

```diff
@@ -232,9 +244,14 @@ Any pod that is using a Rook volume should also remain healthy:
 ## Rook Operator Upgrade Process
 
 In the examples given in this guide, we will be upgrading a live Rook cluster running `v1.4.7` to
-the version `v1.5.0`. This upgrade should work from any official patch release of Rook v1.4 to any
+the version `v1.5.3`. This upgrade should work from any official patch release of Rook v1.4 to any
 official patch release of v1.5.
 
+These methods should work for any number of Rook-Ceph clusters and Rook Operators as long as you
```

Comment:
How about moving this paragraph after the next one? It seems good to mention as early as possible that master builds are not supported.